pcap4j-sample/src/main/java/org/pcap4j/sample/ReadPacketFile.java

```diff
@@ -20,13 +20,13 @@
     = System.getProperty(PCAP_FILE_KEY, "src/main/resources/echoAndEchoReply.pcap");
 
   public static void main(String[] args) throws PcapNativeException, NotOpenException {
-    PcapHandle handle = Pcaps.openOffline(PCAP_FILE);
+    PcapHandle handle = Pcaps.openOfflineWithTstampPrecision(PCAP_FILE, Pcaps.PCAP_TSTAMP_PRECISION_NANO);
```

Comment:
I think it would not be better to merge this change because most libpcap distributions don't support pcap_open_offline_with_tstamp_precision.